other-markdown.Rmd

```diff
@@ -34,93 +42,207 @@ I normally structure my `README` as follows:
 
 4.  An overview that describes the main components of the package.
     For more complex packages, this will point to vignettes for more details.
+    This is also a good place to describe how your package fits into the ecosystem of its target domain.
 
-### README.Rmd {#readme-rmd}
+### `README.Rmd` and `README.md`
 
-If you include an example in your `README` (a good idea!) you may want to generate it with R Markdown.
-The easiest way to get started is to use `usethis::use_readme_rmd()`.
-This creates a template `README.Rmd` and adds it to `.Rbuildignore`.
-The template looks like:
+As mentioned above, we prefer to write `README` in Markdown, i.e. to have `README.md`.
+This will be rendered as HTML and displayed in several important contexts:
 
-    ---
-    output: github_document
-    ---
+-   The repository home page, if you maintain your package on GitHub (or a similar host).
 
-    <!-- README.md is generated from README.Rmd. Please edit that file -->
+    -   [https://github.com/tidyverse/dplyr](https://github.com/tidyverse/dplyr#readme)
 
-    ``` {r, include = FALSE}
-    knitr::opts_chunk$set(
-      collapse = TRUE,
-      comment = "#>",
-      fig.path = "man/figures/README-",
-      out.width = "100%"
-    )
-    ```
+-   On CRAN, if you release your package there.
+
+    -   <https://cran.r-project.org/web/packages/dplyr/index.html>
+
+        Notice the hyperlinked "README" under "Materials".
+
+-   As the home page of your pkgdown site, if you have one.
+
+    -   <https://dplyr.tidyverse.org>
+
+Given that it's best to include a couple of examples in `README.md`, ideally you would generate it with R Markdown.
+That is, it works well to have `README.Rmd` as the main source file, which you then render to `README.md`.
+
+The easiest way to get started is to use `usethis::use_readme_rmd()`.[^other-markdown-1]
+This creates a template `README.Rmd` and adds it to `.Rbuildignore`, since only `README.md` should be included in the package bundle.
+This creates a template `README.Rmd` and adds it to `.Rbuildignore`, since only `README.md` should be included in the package bundle.
+The template looks like this:
+
+[^other-markdown-1]: If it really doesn't make sense to include any executable code chunks, `usethis::use_readme_md()` is very similar to `usethis::use_readme_rmd()`, except that it gives you a basic `README.md` file.
+
+```` markdown
```

Comment:
Like #872 I'm pretty wildly frustrated here. I cannot find any way to have this content (the `README.Rmd` template) included verbatim and have given up for the moment. No matter what I do, when I build the book, I inevitably see:

```
Quitting from lines 112-114 (other-markdown.Rmd) 
Error in library(somepackage) : there is no package called 'somepackage'
Calls: .main ... withVisible -> eval_with_user_handlers -> eval -> eval -> library
```

I.e. somehow it's trying to render the contents of this static verbatim chunk.

Reply:
It feels like none of the usual ways of including verbatim content work in this context. Is it the visual editor or quarto?

Reply:
Even though I link to #872, in this case it has nothing to do with a footnote (which might also be true of #872 ...).

Reply:
I solved this with an explicit verbatim chunk, after a good night of sleep.